custom_components/steam_wishlist/sensor_manager.py

```diff
@@ -137,9 +138,9 @@ async def async_register_component(
 
     @callback
     def async_update_items(self):
-        """Add or remove sensors based on coordinator data."""
+        """Add or update sensors based on coordinator data."""
```

Comment:
This will also remove sensors if you've removed them from your wish list, so let's keep that in the comment.